src/providers/anthropic/chatComplete.ts

```diff
@@ -108,6 +108,13 @@ export const AnthropicChatCompleteConfig: ProviderConfig = {
       required: true,
       transform: (params: Params) => {
         let messages: AnthropicMessage[] = [];
+        if (params.response_format) {
```

Comment:
I know that portkey doesn't list `response_format` as a valid param for Anthropic. Are you sure that it's still on the params object here?

Reply:
Good point; It's available in the `Param` because the `Param` is the general type for all requests. We only need to add it to a message, that's why we check on it in the `messages` config and we do not add another config in the Anthropic one. If Anthropic supports `json_schema`, then we would add another config.